MobiusCore/Source/EffectHandlers/EffectHandler.swift

```diff
@@ -20,29 +20,30 @@
 /// This protocol defines the contract for an Effect Handler which takes `EffectParameters` as input, and produces
 /// `Event`s as output.
 ///
-/// For each incoming effect parameter, zero or more `Event`s can be sent as output using `callback.send(Event)`. When
-/// an the effect parameters have been completely handled, i.e. that effect will not result in any more events, call
+/// For each incoming effect parameter, zero or more `Event`s can be sent as output using `callback.send(Event)`. Once
```

Comment:
This is still weird: "for each incoming effect parameter" - I'm pretty sure most people would think of a parameter set consisting of two Strings as "two effect parameters". 

Better to phrase it as "for each handled Effect", or something like that. Same thing with the sentence below, which makes it sound like each effect parameter (argument! :) ) is handled individually.